src/components/appSwitch/AppSwitch.js

```diff
@@ -106,7 +108,11 @@ const AppSwitch = (props: LoadingProps) => {
     log.debug({ isLoggedIn, isLoggedInCitizen })
     gdstore.set('isLoggedIn')(isLoggedIn)
     gdstore.set('isLoggedInCitizen')(isLoggedInCitizen)
-    isLoggedInCitizen ? API.verifyTopWallet() : Promise.resolve()
+    isLoggedInCitizen
```

Comment:
`initialize` is an async function called `async init()`. Inside `init` we have error handling
So all you have to do - is just to add `await`. Then `API.verifyTopWallet()` promise rejection will be handled by the `init` function:

```javascript
if (isLoggedInCitizen) {
  await API.verifyTopWallet()
}
```

Reply:
You are right - I missed that.
Done